spaceship/lib/spaceship/base.rb

```diff
@@ -59,6 +59,9 @@ def each(&block)
       def to_json(*a)
         h = @hash.dup
         h.delete(:application)
+        if h.key?(:version) && h[:version].kind_of?(Spaceship::Tunes::AppVersion)
```

Comment:
I wonder if we can handle this logic somewhere that is a bit more obvious, like in the app submission itself? Just to make sure it's clear for anyone else coming into this code?